cmd/lncli/cmd_version.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lightningnetwork/lnd/build"
+	"github.com/lightningnetwork/lnd/lnrpc/verrpc"
+	"github.com/urfave/cli"
+)
+
+var versionCommand = cli.Command{
+	Name:     "version",
+	Category: "Info",
```

Comment:
nit: This command now is "alone" in the `Info` category. Should we move `getinfo` and `debuglevel` over too?

Reply:
ended up removing the category, that way these commands are kept near the top of `lncli -l`